suite-common/bluetooth/src/filterOutOldDuplicatesBySameProps.ts

```diff
@@ -0,0 +1,19 @@
+import { BluetoothDeviceCommon } from './types';
+
+// There is a UX need to remap devices that are in pairing mode and emit more than once
+// we do this by filtering out duplicates by name, color and model
+export const filterOutOldDuplicatesBySameProps = <T extends BluetoothDeviceCommon>(devices: T[]) =>
```

Comment:
I'd drop the `BySameProps` since it might be misleading.
```suggestion
export const filterOutOldDuplicates = <T extends BluetoothDeviceCommon>(devices: T[]) =>
```